File: src/test/samples/invalidSamples/invalidSample.clj
;#region FirstRegion
(def x 42)
;;#endregion

;#endregion Invalid end boundary
;#region Invalid start boundary

;;;;   #region Second Region  
(defn my-class []
    ;   #region    InnerRegion   
    (defn my-method [] nil)
    ;;   #endregion    ends InnerRegion  

    ;;   #region  
    (defn my-method2 [] nil)
    ;;#endregion  
)
;; #endregion

File: src/test/samples/invalidSamples/invalidSample.py
#region FirstRegion
x = 42
#endregion
 
  # endregion Invalid end boundary
  # region Invalid start boundary

# region Second Region
class MyClass:
    #  region    InnerRegion    
    def method(self):
        pass
    #          endregion   ends InnerRegion   

    #region
    def method2(self):
        pass
    # endregion

# endregion
